src/github.com/getlantern/yamlconf/yamlconf_test.go

```diff
@@ -214,69 +213,6 @@ func TestCustomPoll(t *testing.T) {
 	}, updated, "Custom polled config should contain correct data")
 }
 
-func TestConfigServer(t *testing.T) {
```

Comment:
The code this test targets was removed at https://github.com/getlantern/lantern/pull/2989